lib/generators/templates/setup/bin_setup.rb

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env ruby
+require "fileutils"
+require "open3"
+
+# path to your application root.
+APP_ROOT = File.expand_path("..", __dir__)
+
+def system!(*args)
+  system(*args, exception: true)
+end
+
+FileUtils.chdir APP_ROOT do
+  if ENV["CI"].nil?
+    puts "\n== Installing Homebrew Bundle from Brewfile =="
+    system! "brew bundle"
+
+    puts "\n== Starting postgresql@15 =="
+    system! "brew services restart postgresql@15"
+
+    puts "\n== Starting redis =="
+    system! "brew services restart redis"
+
+    _, _, status = Open3.capture3("which asdf")
```

Comment:
There are several ways to do this, but I landed on this. I like how explicit it is, but curious what others think.

Reply:
No longer relevant: 97068cd